src/app/groups/[id]/page.tsx

```diff
@@ -19,7 +19,7 @@ import { useAuth } from '@/app/_services/auth-provider';
 // - 각 모달의 열기/닫기, 게시글 CRUD 동작을 연결
 const GroupPage = () => {
   const params = useParams();
-  const groupId = String(params.id);
+  const groupId = String(params.id!);
```

Comment:
next/navigation의 params는 `string | string[]`이기 때문에 

```suggestion
  const groupId = params.id as string;
  
  const groupId = String(params.id); // ! 없이
  
  const isString = (value: unknown): value is string => {
   return typeof value === 'string' || value instanceof String;
 }
  const groupId = isString(id) ? id : id[0];
```
요런 방법들도 있습니다

Reply:
잔소리 아닙니다! 현재 브랜치에서 주제와 맞지 않는 변경사항이 발생하면 현재 브랜치를 타겟하여 pr을 올려 머지 후 현재 브랜치 수정을 진행하면 될까요?

Reply:
가능한 현재 브랜치 주제와 맞지 않는 변경을 하지않는것이 최선이라고 생각하고,
차선으로 말씀해주신대로 현재 브랜치를 타겟하여 pr을 올리는 방법도 있습니다.

## 최선 (PR의 일관성 유지)

1. A 브랜치 작업 
2. B 작업 사항 발생 
3. 작업 중단 
4. 현재까지 작업 사항 A 브랜치에 푸시 
5. A 브랜치에서 develop 타겟으로 PR 생성 
6. 머지됨 
7. develop에서 B 브랜치 생성
8. B 작업

## 차선 (중첩된 PR)

1. A 브랜치 작업 
2. B 작업 사항 발생 
3. 현재까지 작업 사항 A 브랜치에 푸시 
4. A 브랜치에서 develop 타겟으로 PR 생성 
5. A브랜치에서 B 브랜치 생성 
6. B 작업 
7. B 브랜치에 푸시 
8. B 브랜치에서 A 브랜치 타겟으로 PR 생성

다만 이 경우 리뷰어에게 A 브랜치 PR을 먼저 확인한 이후 B 브랜치 PR을 확인해달라고 명시하는것이 좋습니다.

---
최선 전략의 경우 PR간의 연관성이 없어 리뷰 및 히스토리 관리가 용이합니다. 

차선 전략도 PR이 나눠져있어 개별 PR을 리뷰하는데에는 문제가 없지만 PR간의 의존성이 생긴다는것이 문제에요. 

PR간의 의존성이 생긴다는것은 앞선 PR의 맥락을 파악하지 않고는 후속 PR에 대한 리뷰가 어렵다는 의미입니다. 때문에 가능한 최선 전략을 선택하시는걸 권장드려요.

다만 부득이한 이유로 차선을 선택한 경우 리뷰어에게 PR간의 의존성을 잘 설명하고 앞선 PR을 먼저 리뷰하도록 전달해주시면 좋을거같네요.

더 궁금한 사항이 있으시면 언제든지 편하게 말씀해주세요~!